lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -129,7 +136,7 @@ class MailboxDashBoardController extends ReloadableController {
   final accountId = Rxn<AccountId>();
   final userProfile = Rxn<UserProfile>();
   final dashBoardAction = Rxn<UIAction>();
-  final dashboardRoute = DashboardRoutes.thread.obs;
+  final dashboardRoute = Get.arguments?[1] ?? DashboardRoutes.thread;
```

Comment:
If you set the `dashboardRoute` variable to not be an `obx` variable, it may appear wrong on the web. Please test again

Reply:
Shouldn't you just assign a value from `Get.arguments?[1]` like that

Reply:
i have tested it in web, and it works normally